DansTextBasedAdventure.js

```diff
@@ -0,0 +1,70 @@
+const readline = require("readline-sync");
+const chalk = require("chalk");
+let nameInput = readline.question(chalk.yellow("Enter your name:"));
+
+console.log(`Hello ${nameInput}!  Welcome to my game.`);
+
+function startGame() {
+  console.log(chalk.yellow("This is a game in which you open doors until you get a cookie ... or you don't! \n \n "));
+  if (readline.keyInYNStrict("Shall we begin?")) {
+    beginGame();
+  } else {
+    quitGame();
+  }
+}
+
+function beginGame() {
+  console.log(chalk.yellow("Good Choice!! Now to get the cookie, choose from one of 2 doors."));
+  choice1();
+}
+
+function choice1() {
+  let doors = ["Door 1", "Door 2"];
+  console.log(chalk.yellow("Which would you like to open? Hint: It's not the one you just chose..."));
+  answ = readline.keyInSelect(doors);
+  choices = doors[answ];
```

Comment:
you need to declare choices otherwise it becomes global. If it's a singular door think about a better name than choices.